Sources/PostgreSQL/Connection.swift

```diff
@@ -1,5 +1,7 @@
 @_exported import SQL
+import Foundation
 import CLibpq
+import CLibvenice
```

Comment:
Hmm, doesn't look like it. [Venice.poll](https://github.com/Zewo/Venice/blob/8d462a8c625d843cfeced8f6e8358eb64ae9797b/Sources/Venice/Poll/Poll.swift#L20) just wraps `fdwait` - I guess we can use `fdclean` from CLibvenice.